test/Feature/QuerySpec.hs

```diff
@@ -130,7 +130,11 @@ spec =
         [json| [{"text_search_vector":"'baz':1 'qux':2"}] |]
 
     it "matches with computed column" $
-      get "/items?always_true=eq.true" `shouldRespondWith`
+      get "/items?always_true=eq.true&order=id.asc" `shouldRespondWith`
+        [json| [{"id":1},{"id":2},{"id":3},{"id":4},{"id":5},{"id":6},{"id":7},{"id":8},{"id":9},{"id":10},{"id":11},{"id":12},{"id":13},{"id":14},{"id":15}] |]
+
+    it "order by computed column" $
+      get "/items?order=always_true.asc,id.asc" `shouldRespondWith`
```

Comment:
I thought of using ```bizarro_id```, but decided to chose readability over the joke :stuck_out_tongue_winking_eye: